src/Framework/BuildEventArgs.cs

```diff
@@ -144,10 +143,19 @@ public virtual string Message
         /// </summary>
         protected internal string RawMessage
         {
-            get => message;
+            get => FormattedMessage;
```

Comment:
I'm afraid this might be problematic, since RawMessage was explicitly designed to get access to the underlying unformatted message, for both packet translator and binlog reader/writer. Now we're going to realize the formatted message when sending it across the wire.

Reply:
As part of validation for this PR we should ensure that message and arguments remain unrealized when roundtripped across the node packet translator and BuildEventArgsReader/Writer. We should have tests for this, but if not, we should add one for each of both.

Reply:
> Now we're going to realize the formatted message when sending it across the wire.

This actually behaves exactly like before. If the message has been realized it is returned. Otherwise the caller gets the unformatted string. It does not trigger the formatting code - apologies if the names are misleading.

Reply:
Indeed, thanks.